webpack.config.js

```diff
@@ -41,6 +43,7 @@ if (process.env.NODE_ENV === 'production') {
     'webpack-dev-server/client?http://localhost:3000',
     'webpack/hot/only-dev-server'
   ];
+  outputPath = path.join(__dirname, 'app');
```

Comment:
You're right again. Is the contentBase and static server in server/index.js necessary. I guess that's why I used a different path for development

```
const relayServer = new WebpackDevServer(webpack(webpackConfig), {
    contentBase: '/build/',

...

relayServer.use('/', express.static(path.join(__dirname, '../build/app')));
```